src/components/email/Email.tsx

```diff
@@ -93,7 +93,7 @@ export default class Email extends React.Component<
    */
   render(): React.ReactNode {
     return (
-      <Layout {...this.siteConfig}>
+      <Layout {...this.siteConfig} {...this.emailData}>
```

Comment:
we aren't using the entirety of emailData in `<Layout>`, so maybe 
`<Layout {...this.siteConfig} city={this.emailData.city} state={this.emailData.state}>`? if you're feeling ambitious about re-decoupling `<Layout>` from `Email`s, maybe even mutate `siteConfig` in this class and just pass the updated `title` and `meta` as properties ❤️